packages/ra-ui-materialui/src/layout/Notification.js

```diff
@@ -104,6 +104,7 @@ class Notification extends React.Component {
                         </Button>
                     ) : null
                 }
+                classes
```

Comment:
This is most probably not working. Have you tested it ? You're not passing any value to the `classes` prop here, so it's probably considered a boolean

Reply:
Moreover, to avoid warnings from JSS, the `classes` object should probably be cleaned up by removing classes unknown to the Snackbar component (`confirm`, `warning` and `undo`)